rust-mode.el

```diff
@@ -101,6 +101,11 @@ to the function arguments.  When nil, `->' will be indented one level."
   "Face for the question mark operator."
   :group 'rust-mode)
 
+(defface rust-ampersand-face
+  '()
```

Comment:
I think it's common practice to use `'((t :inherit default))`.

Reply:
I've followed your suggestion.